generic_config_updater/gu_common.py

```diff
@@ -152,9 +152,19 @@ def remove_empty_tables(self, config):
         return config_with_non_empty_tables
 
 class DryRunConfigWrapper(ConfigWrapper):
-    # TODO: implement DryRunConfigWrapper
     # This class will simulate all read/write operations to ConfigDB on a virtual storage unit.
-    pass
+    def __init__(self, yang_dir = YANG_DIR):
+        self.yang_dir = YANG_DIR
```

Comment:
>YANG_DIR [](http://example.com/codeflow?start=24&length=8)

Do you mean `yang_dir` instead of `YANG_DIR` ?

Reply:
good catch, removed since base class ctor can be initialized without it